internal/compile-inc/src/test/scala/sbt/inc/IncrementalCompilerTest.scala

```diff
@@ -0,0 +1,254 @@
+package sbt.inc
+
+import java.io.File
+import scala.compat.Platform.EOL
+
+import sbt.IO
+import sbt.Path
+
+import xsbt.{ CompilationFailedException, TestAnalyzingCompiler }
+
+object IncrementalCompilerTest {
+  implicit class FileOP(val content: String)
```

Comment:
Yes, this allows us to write something like `"A.scala" -> delete` to express file deletion.